spec/extensions/fast_fourier_transform_functions.md

```diff
@@ -0,0 +1,670 @@
+# Fast Fourier Transform Functions
+
+> Array API specification for fast fourier transform functions.
+
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
+
+-   Positional parameters must be [positional-only](https://www.python.org/dev/peps/pep-0570/) parameters. Positional-only parameters have no externally-usable name. When a function accepting positional-only parameters is called, positional arguments are mapped to these parameters based solely on their order.
+-   Optional parameters must be [keyword-only](https://www.python.org/dev/peps/pep-3102/) arguments.
+-   Broadcasting semantics must follow the semantics defined in {ref}`broadcasting`.
+-   Unless stated otherwise, functions must support the data types defined in {ref}`data-types`.
+-   Unless stated otherwise, functions must adhere to the type promotion rules defined in {ref}`type-promotion`.
+-   Unless stated otherwise, floating-point operations must adhere to IEEE 754-2019.
```

Comment:
Not for this PR, but: this is getting quite repetitive. We should refactor this, so here it can say something like _"for positional and keyword-only parameters, broadcasting rules, supported data types and type promotion semantics, see XXX"_.

Reply:
I can open a new PR changing this sentence in all the docs :)